sabnzbd/utils/ssdp.py

```diff
@@ -34,9 +41,45 @@ def __init__(self, host, server_name, url, description):
         self.__url = url
         self.__description = description
         self.__myhostname = socket.gethostname()
-        # uuid stays the same as long as hostname and ip address stay the same:
-        self.__uuidXML = uuid.uuid3(uuid.NAMESPACE_DNS, self.__myhostname + self.__host)
-        self.__uuidSSDP = uuid.uuid3(uuid.NAMESPACE_DNS, self.__host + self.__myhostname)
+        # a steady uuid: stays the same as long as hostname and ip address stay the same:
+        self.__uuid = uuid.uuid3(uuid.NAMESPACE_DNS, self.__myhostname + self.__host)
+
+        # descriptionxmlURL = self.__url + "/description.xml"
+
+        # Create the SSDP broadcast message
+        self.__mySSDPbroadcast = f"""NOTIFY * HTTP/1.1
+HOST: 239.255.255.250:1900
+CACHE-CONTROL: max-age=60
+LOCATION: {self.__url}/description.xml
+SERVER: SABnzbd
+NT: upnp:rootdevice
+USN: uuid:{self.__uuid}::upnp:rootdevice
+NTS: ssdp:alive
+OPT: "http://schemas.upnp.org/upnp/1/0/"; ns=01
+
+"""
+        self.__mySSDPbroadcast = self.__mySSDPbroadcast.replace("\n", "\r\n")
+        self.__mySSDPbroadcast = bytes(self.__mySSDPbroadcast, "utf-8")  # convert string to bytes
+
+        # Create the XML info
+        self.__myxml = f"""<?xml version="1.0" encoding="UTF-8" ?>
+<root xmlns="urn:schemas-upnp-org:device-1-0">
+<specVersion>
+<major>1</major>
+<minor>0</minor>
+</specVersion>
+<URLBase>{self.__url}</URLBase>
+<device>
+<deviceType>urn:schemas-upnp-org:device:Basic:1</deviceType>
+<friendlyName>SABnzbd ({self.__myhostname})</friendlyName>
+<manufacturer>SABnzbd Team</manufacturer>
+<manufacturerURL>http://www.sabnzbd.org</manufacturerURL>
```

Comment:
We need to get rid of all the SABnzbd stuff. I would suggest adding 3 extra fields to `__init__`:
```
__init__(self, host, server_name, url, description, manufacturer, manufacturer_url, model)
```
And make the respective `self.__x` variables.